stable/logstash/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.exporter.serviceMonitor.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
Should we add an annotations here?

Reply:
I didn't see annotations from the ones I referenced.
Any suggestions?

Reply:
ping @Aisuko, anything else I can do to help this the changes merged? Thanks

Reply:
@roylee17 Hi. there. Let's assign to the owner first.